src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.aot.cs

```diff
@@ -3,16 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using System.IO;
+using System.Runtime.InteropServices;
 
 namespace System.Diagnostics
 {
     /// <summary>Base class used for all tests that need to spawn a remote process.</summary>
     public abstract partial class RemoteExecutorTestBase : FileCleanupTestBase
     {
-        protected static readonly string HostRunnerName = "xunit.console.netcore.exe";
-        protected  static readonly string HostRunner = TestConsoleApp;
-
-        private static readonly string ExtraParameter = "";
+        protected static readonly string HostRunnerName = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "dotnet.exe" : "dotnet";
```

Comment:
Should we use PlatformDetection instead?

Reply:
https://github.com/dotnet/corefx/blob/master/src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs#L73